Dockerfile.autobuild

```diff
@@ -5,13 +5,34 @@
 # all of it's assets, there are more optimal ways to do this but this is the
 # most friendly and has everything contained in a single instance.
 #
-# Authors: Isaac Bythewood
-# Updated: Mar 29th, 2016
+# Authors: Isaac Bythewood, Jason Kaltsikis
+# Updated: May 2nd, 2020
 # Require: Docker (http://www.docker.io/)
 # -----------------------------------------------------------------------------
 
-# Base system is the LTS version of Ubuntu.
-FROM python:3.7-stretch
+# Build static yarn file
+FROM node:10.20.1-alpine3.11 as yarn-build
+
+WORKDIR pinry-spa
+COPY pinry-spa/package.json .
+RUN yarn install
+COPY pinry-spa .
+RUN yarn build
+
+
+# Required for other database options
+FROM python:3.7-slim-stretch as base
+
+RUN apt-get update \
+    && apt-get -y install gcc default-libmysqlclient-dev
+RUN pip --no-cache-dir install --user mysqlclient cx-Oracle
```

Comment:
It seems that the `psycopg2 ` support has been drop, what's the reason?

Note: `psycopy2` is installed for us to support PostgreSQL. If a user want to use PG, no extra changes id required beside `local_settings.py`.

Reply:
It is because in the Pipfile we install a package psycopg2-binary which is the same as psycopg.